src/screens/CreatePost/index.js

```diff
@@ -125,6 +125,7 @@ const CreatePost = () => {
   const [user] = React.useContext(Context).profile;
   const [allTaggingUser, setAllTaggingUser] = React.useState([]);
   const animatedReminder = React.useRef(new Animated.Value(0)).current;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

An empty line has been added here. This doesn't affect the functionality of the code but helps improve readability by separating different sections of the code.
```diff
+
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->